bindings/src/main/java/io/pravega/storage/gcp/GCPChunkStorage.java

```diff
@@ -169,7 +182,7 @@ protected ChunkHandle doCreateWithContent(String chunkName, int length, InputStr
         BlobId blobId = BlobId.of(this.config.getBucket(), getObjectPath(chunkName));
         BlobInfo blobInfo = BlobInfo.newBuilder(blobId).setContentType("application/octet-stream").build();
         try (InputStream inputStream = data) {
-            this.storage.createFrom(blobInfo, inputStream);
+            Blob blob = this.storage.createFrom(blobInfo, inputStream);
```

Comment:
we are assigning to "Blob blob" variable but i don't see the use of this variable further ?